src/wms/src/test/java/org/geoserver/wms/legendgraphic/JSONLegendGraphicOutputFormatTest.java

```diff
@@ -82,6 +88,16 @@ protected void onSetUp(SystemTestData testData) throws Exception {
         testData.addStyle("dynamicArrows", this.getClass(), catalog);
         testData.addStyle("multiLanguageVector", this.getClass(), catalog);
         testData.addStyle("multiLanguageRaster", this.getClass(), catalog);
+
+        Map<String, String> namespaces = new HashMap<>();
+        //        namespaces.put("xlink", "http://www.w3.org/1999/xlink");
+        //        namespaces.put("xsi", "http://www.w3.org/2001/XMLSchema-instance");
+        //        namespaces.put("wms", "http://www.opengis.net/wms");
+        //        namespaces.put("ows", "http://www.opengis.net/ows");
+        namespaces.put("ogc", "http://www.opengis.net/ogc");
```

Comment:
Are the commented out namespaces needed?

Reply:
Ah nope, I commented the out to see if they were necessary, they were not, but forgot to remove them. Thanks!